### PR TITLE
Adcm-2455_7 add UI tests for policies

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/app/page/admin/page.py [hunk 1]
```diff
@@ -545,42 +544,96 @@ def create_policy(
...
-                else:
-                    raise AssertionError(f"There are no item {item} in select popup")
-
         if users:
             with allure.step(f"Select users {users} in popup"):
                 self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_select)
                 self.wait_element_visible(AdminPoliciesLocators.item)
                 available_users = self.find_elements(AdminPoliciesLocators.item)
-                fill_users_or_group_select(users, available_users)
+                self.fill_select_in_policy_popup(users, available_users)
```
Comment (Sealwing) [inline tests/ui_tests/app/page/admin/page.py:558]: There's no need to open the popup now?

Comment (Sealwing) [reply tests/ui_tests/app/page/admin/page.py:558]: Here and below

File: tests/ui_tests/app/page/admin/page.py [hunk 2]
```diff
@@ -545,42 +544,96 @@ def create_policy(
...
+        self,
+        policy_name: str,
+        description: Optional[str],
+        role: str,
+        users: Optional[str],
+        groups: Optional[str],
+        clusters: Optional[str],
+        services: Optional[str],
+        parent: Optional[str],
+        providers: Optional[str],
```
Comment (Sealwing) [inline tests/ui_tests/app/page/admin/page.py:629]: :octocat:

File: tests/ui_tests/test_admin_page.py [hunk 3]
```diff
@@ -53,6 +73,37 @@ def settings_page(app_fs: ADCMTest) -> AdminSettingsPage:
...
+) -> Tuple[Cluster, Service, Host, Provider]:
+    """Create cluster with component"""
+
+    cluster, service = create_cluster_with_service
+    provider_bundle = sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), 'provider'))
+    provider = provider_bundle.provider_create('test_provider')
+    host = provider.host_create('test-host')
+    cluster.host_add(host)
+    cluster.hostcomponent_set((host, service.component(name=FIRST_COMPONENT_NAME)))
+    return cluster, service, host, provider
```
Comment (Sealwing) [inline tests/ui_tests/test_admin_page.py:104]: For further notice: I suggest keeping one order in return objects.
Cluster -> Service -> Component -> Provider -> Host
That way it'll be easier to read similar code pieces

File: tests/ui_tests/test_admin_page.py [hunk 4]
```diff
@@ -381,11 +440,12 @@ def test_create_policy_on_admin_groups_page(self, app_fs):
             self.custom_policy.role,
             self.custom_policy.users,
             self.custom_policy.groups,
+            None,
+            None,
+            None,
+            None,
```
Comment (Sealwing) [inline tests/ui_tests/test_admin_page.py:448]: maybe make those `None` by default

**Review by Sealwing — CHANGES_REQUESTED** [review]

File: tests/ui_tests/test_admin_page.py (hunk 4 above)
Comment (annademina) [reply tests/ui_tests/test_admin_page.py]: yep, it's much better, done

File: tests/ui_tests/app/page/admin/page.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/app/page/admin/page.py]: added default values

File: tests/ui_tests/app/page/admin/page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/page/admin/page.py:558]: no, popup is opened

Comment (Sealwing) [reply tests/ui_tests/app/page/admin/page.py:558]: Okay, thnx for explanation

**Review by Sealwing — APPROVED** [review]

**Pull request merged**